FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -413,6 +413,49 @@ public async Task<UserRecord> GetUserByPhoneNumberAsync(string phoneNumber)
                 .ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Gets a <see cref="UserRecord"/> object containing information about the user identified by
+        /// <paramref name="providerId"/> and <paramref name="providerUid"/>.
+        /// </summary>
+        /// <param name="providerId">Identifier for the given provider, for example,
+        /// "google.com" for the Google provider.</param>
+        /// <param name="providerUid">The user identifier with the given provider.</param>
+        /// <returns>A task that completes with a <see cref="UserRecord"/> representing
+        /// a user with the specified provider user identifier.</returns>
+        /// <exception cref="ArgumentException">If the provider identifier is null or empty,
+        /// or if the provider user identifier is empty.</exception>
+        /// <exception cref="FirebaseAuthException">If a user cannot be found with the specified
+        /// provider user identifier.</exception>
+        public async Task<UserRecord> GetUserByProviderUidAsync(string providerId, string providerUid)
```

Comment:
Nit: How about calling these `provider` and `providerUid` so that they are not off by just one letter?

Reply:
See also https://github.com/firebase/firebase-admin-node/pull/769#discussion_r374369354.